extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -20,4 +20,8 @@
     border-color: red;
     outline-color: red;
   }
+
+  .form-horizontal > ng-form > .form-control {
```

Comment:
This seems reasonable but is likely to apply too broadly.

Can you prefix all CSS selectors in this file with `fraction-input-interaction` (or whatever the custom html tag is called) so that they only apply within the scope of that interaction?